pkgs/development/python-modules/patator/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, buildPythonPackage, isPy3k, python3Packages, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "patator";
+  version = "0.7";
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.